config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-ci.yaml

```diff
@@ -0,0 +1,21 @@
+periodics:
+  #kubernetes-sigs/cluster-api-provider-vsphere:
```

Comment:
@BenTheElder periodic job does not support repo name,  so i commented it out, but the job still need to checkout the repo, how can i do it?

Reply:
:man_facepalming:
`extra_refs:` -- we can fix this in a follow-up

Reply:
cc me for review and I will re-trigger this with the updated config

Reply:
@BenTheElder  thanks a lot, cced you here: https://github.com/kubernetes/test-infra/pull/11321